src/app/[locale]/exercises/feedback/ThankyouDialog.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+} from '@/components/ui/dialog';
+import {useRouter} from '@/navigation';
+
+import {Confetti} from '@/components/icons/confetti';
+
+interface ThankyouDialogProps {
+  destinationRoute: string;
+}
+
+export default function ThankyouDialog({destinationRoute}: ThankyouDialogProps) {
```

Comment:
The dialog should not be a page. I think the thank you dialog should be a component and used in the Survey component where it's triggered on the button press